core/src/test/java/com/klaytn/caver/common/KeyringContainerTest.java

```diff
@@ -108,118 +111,113 @@ public void checkAddress(List<String> expectedAddress, KeyringContainer containe
-            List<String> expectAddressList = container.generate(10, "entropy");
+            List<String> expectAddressList = caver.wallet.generate(10, "entropy");
 
-            checkAddress(expectAddressList, container);
+            checkAddress(expectAddressList);
         }
     }
 
     public static class newKeyringTest {
+        Caver caver = new Caver(Caver.DEFAULT_URL);
 
         //CA-KEYRINGCONTAINER-003
         @Test
         public void newSingleKeyring() {
-            KeyringContainer container = new KeyringContainer();
-            PrivateKey privateKey = PrivateKey.generate();
+            PrivateKey privateKey = caver.wallet.keyring.generate().getKey();
             String expectAddress = privateKey.getDerivedAddress();
             String expectPrivateKey = privateKey.getPrivateKey();
 
-            AbstractKeyring added = container.newKeyring(expectAddress, expectPrivateKey);
+            AbstractKeyring added = caver.wallet.newKeyring(expectAddress, expectPrivateKey);
             validateSingleKeyring(added, expectAddress, expectPrivateKey);
-            assertEquals(1, container.length());
+            assertEquals(1, caver.wallet.length());
         }
 
         //CA-KEYRINGCONTAINER-004
         @Test
         public void newMultipleKeyring() {
-            KeyringContainer container = new KeyringContainer();
-            String expectAddress = PrivateKey.generate().getDerivedAddress();
+            String expectAddress = caver.wallet.keyring.generate().getAddress();
             String[] expectPrivateKeyArr = {
-                    PrivateKey.generate().getPrivateKey(),
-                    PrivateKey.generate().getPrivateKey(),
-                    PrivateKey.generate().getPrivateKey(),
+                    caver.wallet.keyring.generate().getKey().getPrivateKey(),
+                    caver.wallet.keyring.generate().getKey().getPrivateKey(),
+                    caver.wallet.keyring.generate().getKey().getPrivateKey(),
             };
 
-            AbstractKeyring added = container.newKeyring(expectAddress, expectPrivateKeyArr);
+            AbstractKeyring added = caver.wallet.newKeyring(expectAddress, expectPrivateKeyArr);
             validateMultipleKeyring(added, expectAddress, expectPrivateKeyArr);
         }
 
         //CA-KEYRINGCONTAINER-005
         @Test
         public void newRoleBasedKeyring() {
-            KeyringContainer container = new KeyringContainer();
-            String expectAddress = PrivateKey.generate().getDerivedAddress();
+            String expectAddress = caver.wallet.keyring.generate().getAddress();
             String[][] expectPrivateKeyArr = {
                     {
-                        PrivateKey.generate().getPrivateKey(),
-                        PrivateKey.generate().getPrivateKey(),
-                        PrivateKey.generate().getPrivateKey(),
+                            caver.wallet.keyring.generate().getKey().getPrivateKey(),
```

Comment:
Can you check indentation here?

Reply:
I checked indentations of `core/src/test/java/com/klaytn/caver/common/KeyringContainerTest.java` 
and had some talks with @sirano11 about what indent will we use.

And we agreed to use indent "4" not "8" (even at Continuation Indent).

So I pushed another [commit](https://github.com/klaytn/caver-java/pull/215/commits/e8f08714b4ff88fb8038a0698a20d16daa924b7d) here to apply indent 4 😃

Thanks for feedback @jimni1222